tempesta_fw/t/functional/helpers/control.py

```diff
@@ -162,6 +162,8 @@ def form_command(self):
         # count for remote node.
         if self.threads == -1:
             self.threads = remote.get_max_thread_count(self.node)
+        if self.threads > self.connections:
+            self.threads = self.connections
```

Comment:
If `self.connections` is not multiple of `self.threads`, then `wrk` will establish only `threads * (connections // threads)` connections. The other `connections % threads` connections will be ignored. Is it worth to print warning in this case, what do you think?

Reply:
Good idea, I will add such warning